style.css

```diff
@@ -436,9 +431,195 @@ textarea {
     margin: 0 1.8rem;
   }
 
-  .box2 h1 {
+  .Greetings h1 {
     font-size: 40px;
     max-width: 300px;
     line-height: 3rem;
   }
 }
+
+@media screen and (min-width: 768px) {
+  .box2 {
+    background-image: url(imgs/Header\ bg.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    margin-bottom: 100px;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: 300px 2fr;
+    padding-top: 3rem;
+    margin-bottom: 0;
+  }
```

Comment:
- [ ] I would suggest you to give a width of `40%` to the about myself part of this section and `60%` to the language part in order to match what we have on the figma design. As you can see, your design gives too much width to the language part of the section and that has defeated the purpose of meeting the requirement to produce the close replica of that part of the design. Kindly look into it and fix where the errors are.

### Your Design
![your-about](https://user-images.githubusercontent.com/26597432/206660382-ed29011f-6336-4727-b885-568f0c236fd7.png)

### Figma Design
![figma-about](https://user-images.githubusercontent.com/26597432/206660424-c3764a67-133c-4362-86b7-4da69f189815.png)

- [ ] Also, there's much margin top to the `Get my resume` button which makes it to stand far alone to the rest of the text. Kindly look into it and fix.

![your-get-my-resume-btn](https://user-images.githubusercontent.com/26597432/206660472-1da29bef-2f26-4cb4-8f4a-372874f1f4bf.png)

Reply:
- [ ] As suggested by the previous reviewer, I would encourage you to adjust the width on the `About` and `language` section In order to meet the requirements for this task.

- [ ] I would suggest that you fix the `Get my resume` button on the About myself section to make it look exactly like the Figma design template.

> your design
![errorbtn](https://user-images.githubusercontent.com/71140133/206708163-4ed35350-3d9f-4523-b96b-826e9249beb0.PNG)

> Figma Design Template

![about-btn4](https://user-images.githubusercontent.com/71140133/206708349-87641771-f3c1-4fba-9b9c-3d9d6fde9528.PNG)